molo/profiles/wagtail_hooks.py

```diff
@@ -0,0 +1,71 @@
+from daterange_filter.filter import DateRangeFilter
+from django.contrib.auth.models import User
+from django.http import HttpResponse
+from molo.profiles.admin import ProfileUserAdmin
+from import_export import resources
+from wagtailmodeladmin.options import ModelAdmin, wagtailmodeladmin_register
+from wagtailmodeladmin.views import IndexView
+
+
+class DateFilter(DateRangeFilter):
```

Comment:
Please rename to `FrontendUsersDateRangeFilter` and the template to `frontend_users_date_range_filter.html`